docs/UserGuide.md

```diff
@@ -614,7 +614,14 @@ You can paste the command into the command box after step 2 and repeat step 1-2
 <br>
 
 **Q**: I double-clicked the productivenus.jar file but the app is not starting. What should I do?<br>
-**A**: <br>
+**A**: This is most likely caused by the `security and privacy setting` of your operating system. You can either
+modify your `security and privacy setting` (depends on your operating system) or run ProductiveNUS with the `command
```

Comment:
Security and Privacy Settings can be italicised instead of using a mark up :) Would be good to include "modify your...... to accept anonymous files...."